platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/Messages.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.migration;
+
+import java.util.Optional;
+
+import org.codice.ddf.migration.MigrationException;
+import org.codice.ddf.migration.MigrationMessage;
+import org.codice.ddf.migration.MigrationWarning;
+
+/**
+ * Provides a class for defining messages or message formats for migration exceptions, warnings and
+ * informational messages.
+ */
+public final class Messages {
+    public static final String EXPORT_FAILURE = "Failed to export to file [%s].";
+
+    public static final String IMPORT_FAILURE = "Failed to import from file [%s].";
+
+    public static final String EXPORTING_MIGRATABLE = "Exporting migratable [%s].";
+
+    public static final String IMPORTING_MIGRATABLE = "Importing migratable [%s].";
```

Comment:
❗️ We should not be logging messages that refer to migratables (for this one or the export message on the line above). This is very implementation specific and is going to be confusing to the admin. We are already logging similar messages for us for debugging, so these should be removed.